back-end/lms/models/student.py

```diff
@@ -32,7 +32,56 @@ class Student(Person):
     birthdate = models.DateField()
     program = models.ForeignKey(Program, models.DO_NOTHING)
     status = models.ForeignKey(Status, models.DO_NOTHING)
-    photo = models.CharField(max_length=128, blank=True, null=True)
+    photo = models.ImageField(upload_to = '', max_length=128)
+    place_birth = models.TextField()
+    region = models.TextField()
+    adress = models.TextField()
+    pasport = models.TextField() #можно числовой, но проще так
+    pasport_ufms = models.TextField()
+    pasport_ufms_code = models.TextField()
+    pasport_date = models.DateField()
```

Comment:
I would store these fields in a separate model to reduce the number of fields in the `Student` model. It also makes the data more hierarchical.

Reply:
passport*